### PR TITLE
Issue#655

### PR DESCRIPTION
**Pull request by himanshuc3**
### Expected from this commit
screen shot 2018-02-08 at 8 54 30 am

### Actual
screen shot 2018-02-08 at 8 55 11 am

### CODE REVIEW

File: js/badges.js [hunk 1]
```diff
@@ -25,7 +25,16 @@ function badges() {
         var data = "<li id='badgesItem_" + obj.id + "' class='collection-badges'>" +
             "<img src='" + imageDirectory + obj.image + "' alt='" + obj.name + "' title='" + obj.name + "' class='badgesImg'>" +
             "</li>";
-        $("#badgesDisplay").append(data);
+            $("#badgesDisplay").append(data);
+            $("#badge_count").html("(" + count() + "/4)");
```
Comment (niccokunzmann) [inline js/badges.js]: As an improvement idea from my side. Please add a variable with the number of badges to the badges object and use this. https://en.wikipedia.org/wiki/Magic_number_%28programming%29

File: js/badges.js [hunk 2]
```diff
@@ -25,7 +25,16 @@ function badges() {
...
+    }
+
+
+    function count() {
+        if( typeof count.counter == 'undefined' ) {
+            count.counter = 0;
```
Comment (niccokunzmann) [inline js/badges.js]: How does this refer to https://github.com/fossasia/labyrinth/pull/721/files#diff-836ce988ddaf69630fa37d1807ee2abbR4 ?

**Comment by niccokunzmann** [comment]
@himanshuc3 Thanks for your pull request. What do you think about the comments I left?

File: index.html [hunk 3]
```diff
@@ -231,7 +231,7 @@
         <div class="border1"></div>
 
         <div class="card-content white-text">
-          <span class="card-title"><a class="lang" data-href="badge">Your Badges</a></span>
+          <span class="card-title"><a class="lang" data-href="badge">Your Badges <span id="badge_count"></span></a></span>
```
Comment (niccokunzmann) [inline index.html]: That is cool.
You can use badgeCount, if you like to go in JavaScript style.

**Comment by himanshuc3** [comment]
I did this in a little bit of a hurry. 
Could you answer @niccokunzmann what the items array contained and what was items collision for? 
And thanks for the quick responses.

**Comment by niccokunzmann** [comment]
How about: You collect some badges and have a look what is in it? You can do this with firefox/chrome. Should be `player.badges.items`

**Comment by himanshuc3** [comment]
How about now?
Removed the somewhat static function `count()`.

**Comment by niccokunzmann** [comment]
This is ok for me now. This one is still remaining:



> As an improvement idea from my side. Please add a variable with the number of badges to the badges object and use this. https://en.wikipedia.org/wiki/Magic_number_%28programming%29

**Comment by himanshuc3** [comment]
@niccokunzmann That could be a bit redundant according to me as then it would have to be updated everywhere items array is updated. I think this does it in a little less lines.

**Pull request merged**